spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -867,10 +885,11 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
 
     withSQLConf(
       CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true",
+      CometConf.COMET_CAST_ALLOW_INCOMPATIBLE.key -> "true",
       CometConf.COMET_SHUFFLE_MODE.key -> "native") {
       Seq(true, false).foreach { dictionaryEnabled =>
         withSQLConf("parquet.enable.dictionary" -> dictionaryEnabled.toString) {
-          val table = "t1"
+          val table = s"final_decimal_avg_$dictionaryEnabled"
```

Comment:
These changes ended up not being entirely necessary, but the test did have a mix of hard-coded `t1` and use of the variable `$tableName` references in SQL, and I made these consistent.